src/main/java/bio/terra/app/configuration/ApplicationConfiguration.java

```diff
@@ -370,7 +371,8 @@ public ObjectMapper objectMapper() {
         return new ObjectMapper()
             .registerModule(new ParameterNamesModule())
             .registerModule(new Jdk8Module())
-            .registerModule(new JavaTimeModule());
+            .registerModule(new JavaTimeModule())
+            .configure(MapperFeature.ACCEPT_CASE_INSENSITIVE_VALUES, true);
```

Comment:
This seems like a big change to the default objectmapper used in tons of places. Is there another way to do this that doesn't affect all other uses?